src/main/java/cc/polyfrost/oneconfig/renderer/asset/Icon.java

```diff
@@ -57,25 +58,25 @@ public Image getImage() {
 
     public void draw(long vg, float x, float y, float width, float height, int color, float scale) {
         if (type == Type.SVG) {
-            RenderManager.drawSvg(vg, getSVG(), x, y, width, height, color, scale);
+            NanoVGHelper.INSTANCE.drawSvg(vg, getSVG(), x, y, width, height, color, scale);
         } else if (type == Type.IMAGE) {
-            RenderManager.drawImage(vg, getImage(), x, y, width, height, color);
+            NanoVGHelper.INSTANCE.drawImage(vg, getImage(), x, y, width, height, color);
```

Comment:
ic

Reply:
this is actually the right use of this, there's no need to store a NanoVGHelper instance since it's gonna be used once either way because of the if statement